src/OneExplorer/ConfigObject.ts

```diff
@@ -18,6 +18,7 @@ import * as fs from 'fs';
 import * as ini from 'ini';
 import * as path from 'path';
 import * as vscode from 'vscode';
+import {Logger} from '../Utils/Logger';
 
 /**
```

Comment:
@YongseopKim I defined the term `Artifact` here.

Reply:
Most of times, `artifacts` are a zip file including all the result output files, but here, artifact means each result file, I think that's what you are talking about... I took a reference [here](https://en.wikipedia.org/wiki/Artifact_(software_development)), wikipedia, it doens't mention the notion as  'a set of results files', so I assumed that the term artifact can be used to each file.

> The term artifact in connection with software development is largely associated with specific development methods or processes e.g., [Unified Process](https://en.wikipedia.org/wiki/Unified_Process). This usage of the term may have originated with those methods.